routes/giffy.js

```diff
@@ -155,10 +155,54 @@ giffyRouter.post('/logout', (req, res) => {
 });
 // Georgia credit:
 // delete a single favorite from the user's favorites
+giffyRouter.post('/delFavorite', async (req, res) => {
+  try {
+    const { userId, itemId } = req.body;
+    const result = await query(
+      'DELETE FROM favorites (user_id, item_id) VALUES (?, ?)',
+      [userId, itemId]
+    );
+    if (result.affectedRows === 1) {
+      console.log("Removed from Favorites");
```

Comment:
```suggestion
      console.log(`Delete, userId:${userId}, itemId${itemId}`);
```
I like to think if we're logging to the console, one day we'll actually be able to derive business value from it.